### ISSUE TITLE
Homepage - UI

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#23]
## User story
1. As a user
2. I want / need to see a homepage when I first access the shared desk management system before logging in
3. So that I can learn what Deskstar is, what features it provides and access the documentation

## Acceptance criteria
* Homepage should have a nice UI and be the first page a logged out user sees when accessing our system
* Homepage should describe what Deskstar is and what basic functionality it provides
* Homepage should link or contain a documentation in case users have problems or want to know more before registering
* Homepage should provide access to the login and registration function, which should be immediatly visible

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
